soundbay/trainers.py

```diff
@@ -70,7 +72,8 @@ def train(self):
             self.logger.reset_losses()
             self.train_epoch(epoch)
             self.epochs_trained += 1
-            self.eval_epoch(epoch)
+            self.eval_epoch(epoch, 'val')
+            self.eval_epoch(epoch, 'train_as_val')
```

Comment:
perhaps we could add flag for `eval_train_as_val_interval` so we won't have to go over the whole train set every epoch, because it might be much larger than the validation set. I think a default of once per 10 epoch could suffice, wdyt?